assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -59,10 +59,6 @@ group.rcuc=0:f:11:*:rcuc.*
 group.ktimers=0:f:11:*:ktimers.*
 sched_min_granularity_ns=10000000
 sched_migration_cost_ns=5000000
-numa_balancing=0
-{{if .RealTimeHint}}
-sched_rt_runtime_us=-1
```

Comment:
do we want to remove also this one?

Reply:
If you mean `sched_rt_runtime_us`, then yes.  There is no such `[scheduler]` directive.

Reply:
yes, this is what I meant. Looks like we need to do a better job verifying these directives when reviewing future PRs.

Reply:
I think the test needs to be updated to expect this modified content

Reply:
Thanks, adjusted based on a rendered `Tuned` CR.